### DISCUSSION TITLE
[Support Request] Request dkms support for Rtl8761b kernel module

### DISCUSSION THREAD

**Discussion by Dvalin21** [home-assistant/operating-system#3017] [Ideas]
Requesting added support for RTL8761b to add use of 5.1+ bluetooth adapters. 
The following is dkms reference https://forums.debian.net/viewtopic.php?t=155643

**Comment by sairon** [comment]
Home Assistant OS (or Buildroot respectively) does not have a way for DKMS mechanism to work - all binaries are compiled as a part of the system image. However, this device should be supported by the upstream Linux kernel driver. If it does not work for you, please create an issue in the repository issue tracker with host logs and full details about the hardware.
